core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/CostBasedRegionPlanGenerator.scala

```diff
@@ -33,7 +33,13 @@ class CostBasedRegionPlanGenerator(
 
   def generate(): (RegionPlan, PhysicalPlan) = {
 
+    val startTime = System.nanoTime()
     val regionDAG = createRegionDAG()
+    val schedulerTime = System.nanoTime() - startTime
+    logger.info(
+      s"WID: ${workflowContext.workflowId.id}, EID: ${workflowContext.executionId.id}, total RPG time: " +
```

Comment:
The logger context should already have wid and execution id information. Can you check?

Reply:
The logger context does not have workflow context info. How would actor logging contain workflow context?

Reply:
We customized actor logging to have those information. You can see most of the logs have worker id printed. If you don't see it for this log message, then you are probably using lazylogging. See if it is easy to switch to actor logging, if it is hard, then nvm.

Reply:
Switched to AmberLogger.